cmd/root.go

```diff
@@ -79,13 +80,21 @@ func Execute() error {
 		if len(GitBranch) == 0 {
 			GitBranch = "detached HEAD"
 		}
+
 		annotation["version"] = GitTagVersion
 		annotation["branch"] = GitBranch
 		annotation["commitInfo"] = GitCommitInfo
-		rootCmd.SetVersionTemplate(`ghpc version {{index .Annotations "version"}}
+		tmpl := `ghpc version {{index .Annotations "version"}}
 Built from '{{index .Annotations "branch"}}' branch.
 Commit info: {{index .Annotations "commitInfo"}}
-`)
+`
+		tfVersion, _ := shell.TfVersion()
+		if tfVersion != "" {
```

Comment:
nit: Wouldn't not err be cleaner.

Reply:
I'm on a fence about this, we can make it more verbose later.